test_conformance/api/test_queries.cpp

```diff
@@ -476,30 +476,124 @@ REGISTER_TEST(get_command_queue_info_compatibility)
 REGISTER_TEST(get_context_info)
 {
     int error;
+
+    // query single device context and perform object comparability test
+    clContextWrapper context_to_test =
+        clCreateContext(NULL, 1, &device, nullptr, nullptr, &error);
+    test_error_ret(error, "Unable to create testing context", CL_SUCCESS);
```

Comment:
Creating a context is relatively expensive.  Could we query the device from the context provided by the harness instead of creating a new context?

Reply:
Corrected.